### PR TITLE
Lambda versioning

### PR DESCRIPTION
**Pull request by musketyr**
* fixed publishing for functions which already exists
* ability to assign an alias to the published version

### CODE REVIEW

File: src/main/java/jp/classmethod/aws/gradle/lambda/AWSLambdaMigrateFunctionTask.java [hunk 1]
```diff
@@ -23,6 +23,8 @@
 import java.nio.channels.FileChannel;
 import java.util.Map;
 
+import com.amazonaws.services.lambda.model.*;
```
Comment (dai0304) [inline src/main/java/jp/classmethod/aws/gradle/lambda/AWSLambdaMigrateFunctionTask.java]: Avoid using wildcards please

Comment (musketyr) [reply src/main/java/jp/classmethod/aws/gradle/lambda/AWSLambdaMigrateFunctionTask.java]: sure. it's fixed in the following commit.

**Comment by dai0304** [comment]
Please fix CI build.
We have import ordering rules.

**Comment by musketyr** [comment]
@dai0304 can you give me a hint? I'm not quite sure how to apply the rule.

**Comment by dai0304** [comment]
Run `./gradlew spotlessApply` and commit :)

**Comment by musketyr** [comment]
thanks, @dai0304, that did the job :-) I need to take a look at that Spotless tool, looks very handy. sorry for spinning your Circle CI that much :-)

**Comment by frankfarrell** [comment]
@musketyr I just noticed this PR, looks useful and is related to this PR too: https://github.com/classmethod/gradle-aws-plugin/pull/133/files

**Pull request merged**